docs/docs/concepts/flows.mdx

```diff
@@ -460,23 +460,44 @@ If the "final_confirmation" slot is already filled, the bot will clear it, ask
 the `collect_information` step and fill the slot again. This ensures the confirmation
 `collect_information` step is always asked.
 
-#### Scopes
+#### Reset slot value after flow execution
 
-Scopes determine the lifespan of a slot or variable. There are two types of
-scopes available:
+You can determine whether to reset the slot value after the flow execution or not by setting the optional
+`reset_after_flow_ends` property to `true` or `false` respectively. If the `reset_after_flow_ends` property is set to
+`True`, the slot value exists only for the duration of the flow. Once the flow ends, the slot value is either reset to
+`null` or the slot's initial value (if provided in the domain file under `slots`). If instead the `reset_after_flow_ends`
+property is set to `false`, this indicates that the slot value should persist across flows until it is explicitly changed.
 
-- **flow**: The variable exists only for the duration of the flow. Once the flow
-  ends, the variable ceases to exist.
-- **global**: The variable persists across flows until it is explicitly changed.
-
-The slot can also be configured to be cleared at the end of the flow execution.
 The following configuration would clear the answer to the `collect_information` step when a
 flow is completed.
 
 ```yaml
 - id: "another_confirmation_step"
-  collect_information: "yes_or_no" # slot name
-  scope: "flow"
+  collect_information: "my_custom_slot" # slot name
+  reset_after_flow_ends: true
+```
+
+If the property is not defined for the `collect_information` step, the default value will be `true`.
```

Comment:
I would put this at the top of this section, so that people can quickly read about the default behaviour. I actually missed this in my first read.